TRIAGING.md

```diff
@@ -12,19 +12,29 @@ You can track if your issue was triaged by watching your GitHub notifications fo
 
 Each meeting we seek to address all new issues. However, should we run out of time before your issue is discussed, you are always welcome to attend the next meeting or to follow up on the issue post itself.
 
-### How do I join the Triage meeting?
+### How many triage meetings are there?
 
-Meetings are hosted regularly at 10:00a - 10:55a Central Time every Wednesday and can be joined via [Zoom](https://zoom.us/download), with this [meeting link](https://us02web.zoom.us/j/86287450465) and passcode `805212`.
+The OpenSearch repository is very large and as a result, the rate of incoming issues and technical expertise required is too large for any one person to be able to effectively address all issues. Therefore, we have multiple triage meetings scoped to the following component areas: Search, Storage, Cluster Manager, and finally "Core" as a catch-all for all other issues. Component labels are used to filter issues, and each triage meeting will use these labels to construct a search as follows to target issues:
```

Comment:
I don't think attendees are interested in 'why' they just want to know the answer to the question. What do you think of the following?
```suggestion
There are several weekly triage meetings scoped to the following component areas: Search, Storage, Cluster Manager, and finally "Core" as a catch-all for all other issues.  Depending on the issue's labels will determine which triage meeting will review the issue.
```

I think the question `How do I join the Triage meeting?` should still be first, the answer just got a little more interesting with many potential meetings.

Reply:
Totally agree the "why" isn't necessary and I like the shorter text, but I will not be taking your two-spaces-after-the-period suggestion

Reply:
I actually moved the "There are several weekly triage meetings..." line to the top paragraph. Let me know what you think.